ftw/simplelayout/aliasblock/browser/templates/aliasblock.pt

```diff
@@ -0,0 +1,3 @@
+<div:aliasblock class="sl-alias-block">
+  <h2 tal:content="view/block_title" tal:condition="view/block_title">Title</h2>
```

Comment:
Add condition for the mentioned `show_title` field.

Reply:
This is removed in further implementation. There is no title in this block as specified.